Subject: [PATCH] ignore empty test results as they are not generated for no changes
---
Index: cmake/config.cmake.in
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cmake/config.cmake.in b/cmake/config.cmake.in
--- a/cmake/config.cmake.in	(revision bf5b866ff4544d2b8c18b9ba249fc79cfb1da4b1)
+++ b/cmake/config.cmake.in	(date 1778050229238)
@@ -3,6 +3,7 @@
 set(_@PROJECT_NAME@_supported_components @package_components@)
 set(_@PROJECT_NAME@_georges_dependencies @package_georges_dependencies@)
 set(_@PROJECT_NAME@_gui_dependencies @package_gui_dependencies@)
+set(_@PROJECT_NAME@_misc_dependencies @package_misc_dependencies@)
 set(_@PROJECT_NAME@_sound_dependencies @package_sound_dependencies@)

 foreach(_comp ${@PROJECT_NAME@_FIND_COMPONENTS})
@@ -22,7 +23,7 @@


 # Dependencies for components
-foreach(_comp ${_@PROJECT_NAME@_supported_components})
+foreach(_comp ${@PROJECT_NAME@_FIND_COMPONENTS})
   foreach(_dependency IN ITEMS ${_@PROJECT_NAME@_${_comp}_dependencies})
     find_dependency(${_dependency} REQUIRED)
   endforeach()
Index: nel/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/nel/CMakeLists.txt b/nel/CMakeLists.txt
--- a/nel/CMakeLists.txt	(revision bf5b866ff4544d2b8c18b9ba249fc79cfb1da4b1)
+++ b/nel/CMakeLists.txt	(date 1778050295766)
@@ -95,6 +95,7 @@
 set(package_required_dependencies CURL Freetype)
 set(package_georges_dependencies LibXml2)
 set(package_gui_dependencies Lua Luabind)
+set(package_misc_dependencies LibXml2 ZLIB JPEG GIF)
 set(package_sound_dependencies Vorbis)


